test/gui/shared/steps/steps.py

```diff
@@ -380,6 +387,11 @@ def step(context, username, foldername):
     createFolder(context, foldername, username)
 
 
+@When('user "|any|" tries to create a folder "|any|" inside the sync folder')
```

Comment:
I don't think we need an extra step here, that does the same thing as the one above?

actually the user does not only try to create a folder, the user creates a folder (on the local disk) but it might or might not get synced to the server.

Same for the steps with files